core/src/scss/components/button/_button.scss

```diff
@@ -12,12 +12,24 @@
 //
 // Style guide: Simple.Button
 //
+button {
+  &,
+  &.su-button {
+    &,
+    &.su-link {
+      @include button-primary;
+    }
+  }
+}
+
 .su-button,
-.su-button.su-link,
-button,
 [type='button'],
 [type='submit'],
 [type='reset'],
 [type='image'] {
-  @include button-primary;
+  &,
+  &.su-link {
+    @include button-primary;
```

Comment:
And here :) 

@cjwest Do please double check, but I believe we have `<a>` with the `.su-button` and `.su-link` classes, but I don't think we have `<button>` with `.su-link` class.